_non-cosmos/ethereum/assetlist.json

```diff
@@ -129,6 +129,15 @@
       "name": "Dai Stablecoin",
       "display": "dai",
       "symbol": "DAI",
+	    "traces": [
+        {
+          "type": "synthetic_CDP",
+          "platform": "forex",
```

Comment:
What does platform mean?

Reply:
I'm generally confused why we have pricing info in the assetlist, though maybe I'm missing some of the vision for who the assetlist serves

Reply:
"platform" refers to the place an assets exists--usually a blockchain (e.g., Ethereum), or perhaps an exchange (e.g., NYSE). "platform" and "base_denom" together form a key used to identify any asset. The platform part is precisely used to identify which folder in the Chain Registry the asset is listed, so it must be spelled exactly as its folder name appears. The base_denom part is then used to search for the asset within that other platform/folder's assetlist.json file.

"forex" is just the platform name I selected (open to other suggestions) to refer to real-world government currencies. In the case of DAI, since it claims to follow the price of the U.S. Dollar, the platform "forex" and base_denom "USD" is how the registry tells us this. So, the object within the traces array here is not representing DAI--but instead the U.S. Dollar. Thus, the platform and base_denom therein should be "forex" and "USD", respectively. Whereas, in the case of stETH, it's not meant to follow the price of the US Dollar, it's meant to follow the price of ether, hence why it's origin has base_denom as "wei".

I hope this makes it more clear.

Reply:
Uhh why is this telling us about what the price it tracks is. This feels like a very awkward way of describing pegged vs non pegged coins

I'd expect platform for Dai to be ethereum